server/game/cards/14.2-SS/SpectralVisitation.js

```diff
@@ -2,21 +2,21 @@ const ProvinceCard = require('../../provincecard.js');
 const AbilityDsl = require('../../abilitydsl');
 const { CardTypes, Locations, Players, Durations, Phases } = require('../../Constants');
 
-const spectralVisitationCost = () => ({
-    action: { name: 'spectralVisitationCost', getCostMessage: () => ['discard the top 4 dynasty cards', []] },
-    canPay: function (context) {
-        return context.player.dynastyDeck.size() >= 4;
-    },
-    resolve: function(context) {
-        context.costs.spectralVisitationCost = context.player.dynastyDeck.first(4);
-    },
-    pay: function(context) {
-        const discardedCards = context.costs.spectralVisitationCost;
-        discardedCards.slice(0, 4).forEach(card => {
-            card.controller.moveCard(card, Locations.DynastyDiscardPile);
-        });
-    }
-});
+// const spectralVisitationCost = () => ({
```

Comment:
Can you delete the code rather than commenting it out? We can always find it in the git history if we need.

Reply:
oops, just forgot